package.json

```diff
@@ -63,7 +63,7 @@
     "supertest": "^6.0.1",
     "uuidv4": "^6.2.4"
   },
-  "husky": {
+  "_husky": {
```

Comment:
Please do not simply deactivate husky. Please check out [CONTRIBUTING.md#other-thingies](https://github.com/sapmentors/cds-pg/blob/main/docs/CONTRIBUTING.md#other-thingies) how to format commit messages to be OK for the check.

Reply:
> CONTRIBUTING.md#other-thingies

hi @gregorwolf 

the pr was only meant as pointer, not as contribution ;)

tests are running now (cf. require of deploy), but an expect is not fulfilled anymore...